definitions/features/hammer.rb

```diff
@@ -28,11 +28,11 @@ def config_directories
   def setup_admin_access
     return true if check_connection
     logger.info('Hammer setup is not valid. Fixing configuration.')
-    custom_config = { :foreman => { :username => 'admin' } }
+    custom_config = { :foreman => { :username => admin_username } }
     custom_config = on_invalid_host(custom_config)
     custom_config = on_missing_password(custom_config) # get password from answers
     custom_config = on_invalid_password(custom_config) # get password from answers
-    ask_password_and_check(custom_config) unless ready?
+    ask_username_password_and_check(custom_config) unless ready?
```

Comment:
Do we really need to ask for username?
As foreman-maintain first checks connection using hammer credentials, if that fails to connect it tries to connect using values from installer answers file.

@mbacovsky, any thoughts about this change?

Reply:
@kgaikwad we can remove it and just keep the message saying to check whether the username is correct in `#{custom_config_file})`.

Reply:
@kgaikwad I have removed this change.